sdk/cosmosdb/cosmos/src/queryExecutionContext/documentProducer.ts

```diff
@@ -50,6 +53,9 @@ export class DocumentProducer {
     targetPartitionKeyRange: PartitionKeyRange,
     options: FeedOptions,
     correlatedActivityId: string,
+    startEpk?: string,
+    endEpk?: string,
+    populateEpkRangeHeaders?: boolean,
```

Comment:
Is it possible for startEpk/endEpk to be passed to the document producer but not used?

Reply:
Yes it  is possible for startEpk/endEpk to be passed to the document producer but not used. Intially when we insert document producers to the priority queue we pass the start and end epk headers and populateEpkRangeHeaders = false. So until a merge happens,  we dont use the start and end epk headers. If a merge happens then we use the parent document producer's start and end epk headers.